ext/pdo_sqlite/tests/gh13991.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Fix GH-13984: Buffer size is now checked before memcmp
+--EXTENSIONS--
+pdo_sqlite
+--FILE--
+<?php
+$dbfile = 'z.db';
+$db = new PDO('sqlite:' . $dbfile, null, null, [PDO::ATTR_PERSISTENT => true]);
+echo 'done!';
+?>
+--CLEAN--
+<?php
+@unlink(getcwd() . '/z.db');
```

Comment:
Is there a better way?

Reply:
We should be careful _somehow_ that we don't destroy files or users running the tests.

Reply:
Changed to skip if a file with the same name already exists.